### PR TITLE
Added indexed property

### PR DESCRIPTION
**Pull request by marjasi**

### CODE REVIEW

File: ShopLens/ShopLensForms/Views/MainForm.cs [hunk 1]
```diff
@@ -11,21 +11,19 @@ public partial class ShopLens : Form
     {
         private FilterInfoCollection _captureDevices;
         private VideoCaptureDevice _videoSource;
-        private MainController _mainController;
+        public MainController _mainController { get; set;}
```
Comment (EdvardasDlugauskas) [inline ShopLens/ShopLensForms/Views/MainForm.cs]: Naming should be CamelCase

File: ShopLens/ShopLensForms/Views/IntroForm.cs [hunk 2]
```diff
@@ -6,14 +6,12 @@ namespace ShopLensForms
 {
     public partial class IntroForm : Form
     {
-        private MainController _mainController;
+        public MainController _mainController { get; set; }
```
Comment (EdvardasDlugauskas) [inline ShopLens/ShopLensForms/Views/IntroForm.cs]: Naming should be CamelCase

**Comment by EdvardasDlugauskas** [comment]
Reviewed during pair-programming session.

**Review by EdvardasDlugauskas — APPROVED** [review]

File: ShopLens/ShopLensForms/Views/MainForm.cs [hunk 3]
```diff
@@ -11,21 +11,19 @@ public partial class ShopLens : Form
     {
         private FilterInfoCollection _captureDevices;
         private VideoCaptureDevice _videoSource;
-        private MainController _mainController;
+        public MainController MainController { get; set;}
```
Comment (TomasAdomavicius) [inline ShopLens/ShopLensForms/Views/MainForm.cs:14]: Since MainController is public, do we still need get, set methods?

Comment (marjasi) [reply ShopLens/ShopLensForms/Views/MainForm.cs:14]: Imma make it private then.

File: ShopLens/ShopLensForms/Views/IntroForm.cs [hunk 4]
```diff
@@ -6,26 +6,24 @@ namespace ShopLensForms
 {
     public partial class IntroForm : Form
     {
-        private MainController _mainController;
+        public MainController MainController { get; set; }
```
Comment (TomasAdomavicius) [inline ShopLens/ShopLensForms/Views/IntroForm.cs:9]: Change modifier to private

**Pull request merged**